### PR TITLE
Corrects JoinKeys vignette and adds merge section

### PR DESCRIPTION
**Pull request by averissimo**
# Pull Request

### Changes description

Suggestions from talk with @vedhav

### CODE REVIEW

**Review by vedhav — APPROVED** [review]
Thanks LGTM

**Comment by averissimo** [comment]
@kartikeyakirar should this be merged to `pre-release` branch or should it target `main`?

**Comment by kartikeyakirar** [comment]
> should this be merged to pre-release branch or should it target main?

its a part of the cleanup process, I believe it would be beneficial to merge it with the pre-release.

File: vignettes/join-keys.Rmd [hunk 1]
```diff
@@ -53,8 +53,8 @@ As the _Constructor_ it is used to specify a collection of multiple individual k
 Specifies a primary key or a relationship between two datasets.
 
 - `dataset_1`, `dataset_2`: names of the datasets (if `dataset_2` is the same as `dataset_1` or is omitted, it creates a primary key);
-- `key` (optional): named vector of column names;
-- `parent` (optional): indicates which dataset (`"dataset_1"` or `"dataset_2"`) is the parent in a _parent-child_ relationship, or `"none"` for an undirected relationship.
+- `key`: named character vector of column name mapping between datasets (unnamed vector assumes column names between datasets are the same);
```
Comment (kartikeyakirar) [inline vignettes/join-keys.Rmd:56]:
```suggestion
- `key`: named `character` vector of column name mapping between datasets (unnamed vector assumes column names between datasets are the same);
```

**Pull request merged**